community/zsh/depends

```diff
@@ -0,0 +1,3 @@
+make make
+xz make
```

Comment:
`xz` should go after `ncurses` (alphabetical ordering is desired though it sometimes makes sense to order things differently).